src/sass/utils/_variables.scss

```diff
@@ -291,6 +291,32 @@ $nav-pills-active-link-hover-bg    : $component-active-bg;
 $nav-pills-active-link-hover-color : $component-active-color;
 
 // Pagination
+$line-height-lg:         (4 / 3) !default;
+$line-height-sm:         1.5 !default;
+
+$border-radius:          .25rem !default;
```

Comment:
I'm not sure how it's done elsewhere but maybe check if we usually have leading zeroes on the decimals.